packages/yourloops/test/integration/pages/patient-dashboard/patient-dashboard.hcp.spec.tsx

```diff
@@ -151,43 +160,41 @@ describe('Patient dashboard for HCP', () => {
   })
 
   it('should be able to use chat widget', async () => {
-    mockDataAPI(completeDashboardData)
-
     await act(async () => {
       renderPage(patientDashboardRoute)
     })
 
     await testChatWidgetForHcp()
   })
 
-  it('should render correct statistic when data is two weeks old', async () => {
+  it('should render correct statistic when data is  old', async () => {
```

Comment:
```suggestion
  it('should render correct statistic when data is two weeks old', async () => {
```
Try having the same test name than the one you used for other roles (here I suggest to reuse the test name that you used for the patient role).
However, there is something weird with this test, you mock the data for 16 days but you call the method `testDashboardDataVisualisationWithOldData` that you usually call for when the data is mocked for 14 days. This is acceptable (even though you have a method named `testDashboardDataVisualisationTwoWeeksOldData`), however, the test afterward is called `should produce fourteen days old statistics when data is sixteen days old` in which you mocked the test with 14 days old data and not 16... There is a mixup, fix it